### ISSUE TITLE
Update node requirement

### ISSUE THREAD

**Issue by mmmavis** [mozilla/foundation.mozilla.org#4134]
Our engine requirement in `package.json` is currently
```js
{
  ...
  "engines": {
    "node": "^8.9.0",
    "npm": "^6.0.0"
  }
  ...
}
```

As some of the node packages Foundation site uses have started to terminate support for node v8 (e.g., [`postcss-cli`](https://github.com/postcss/postcss-cli/blob/master/CHANGELOG.md#700--2020-01-07)), we should probably update our [node](https://nodejs.org/ca/blog/release/) requirement.

**Comment by Pomax** [comment]
closed by #4191

**Comment by mmmavis** [comment]
Yay!
